modules/node_modules/@ncigdc/uikit/Banner.js

```diff
@@ -9,8 +9,9 @@ const style = {
     display: 'flex',
     alignItems: 'center',
     backgroundColor: '#265986',
-    color: 'white',
+    color: '#000000',
```

Comment:
oh wait, yellow background? The different notification levels have different bg colours, change the foreground colour for yellow here https://github.com/NCI-GDC/portal-ui/blob/2c78e1df5be20c81b7b137c81e302ba1571e392d/modules/node_modules/%40ncigdc/uikit/Banner.js#L37 otherwise
<img width="497" alt="screen shot 2017-05-19 at 12 01 27 pm" src="https://cloud.githubusercontent.com/assets/1314446/26256487/1b010df8-3c8b-11e7-968d-9c9c65c26328.png">